app/src/test/java/org/oppia/android/app/testing/PlatformParameterIntegrationTest.kt

```diff
@@ -90,10 +123,26 @@ class PlatformParameterIntegrationTest {
     )
   }
 
+  private val mockPlatformParameterListWithToastDisabled by lazy {
+    val mockSplashScreenWelcomeMsgParam = PlatformParameter.newBuilder()
+      .setName(SPLASH_SCREEN_WELCOME_MSG)
+      .setBoolean(SPLASH_SCREEN_WELCOME_MSG_DEFAULT_VALUE)
+      .build()
+
+    listOf<PlatformParameter>(
```

Comment:
Can this fit on a a single line?
```suggestion
    listOf<PlatformParameter>(mockSplashScreenWelcomeMsgParam)
```

Reply:
Done